tools/src/launcher/Main.hs

```diff
@@ -280,7 +281,7 @@ main =
     setEnv "LC_ALL" "en_GB.UTF-8"
     setEnv "LANG"   "en_GB.UTF-8"
 
-    LO {..} <- getLauncherOptions
+    lo@LO {..} <- getLauncherOptions
```

Comment:
'RecordWildCards' will be gone..
can the field accessed in LO be named?

Reply:
@cleverca22 I would actually leave that as it is for now.

What we are currently doing is moving `RecordWildCards` from the default language extensions to a `LANGUAGE` pragma at the top of the files that need it.

While that is being done we are recording how RCWs are being used and removing the ones that are the most difficult to read (ie using RCW for types in an upstream package).

In this case `LauncherOptions` is in the same package and hence probably does not need to change.

Reply:
yeah, the code was also looking much worse after removing it, so i put it back in